compiler/cpp/src/thrift/generate/t_js_generator.cc

```diff
@@ -510,6 +511,26 @@ string t_js_generator::render_includes() {
   return result;
 }
 
+/**
+ * Renders all the imports necessary for including another Thrift program
```

Comment:
this comment seems a bit strange: do you mean "renders all of the imports necessary to allow this file to be included by other files"?

Reply:
No, assume there are 2 thrift files `A.thrift` and `B.thrift`.
B includes A: `include "A.thrift"`
This outputs to `B.d.ts` the following:
```ts
import A_ttypes = require('./A_types');
```

Reply:
I think a single Thrift file is considered a Thrift program when generating code.